app/services/remark_generators/frequency_checker.rb

```diff
@@ -2,7 +2,12 @@ module RemarkGenerators
   class FrequencyChecker < BaseChecker
     include Exemptable
 
-    def call
+    def self.call(assessment, collection, date_field = "payment_date")
+      new(assessment, collection).call(date_field)
+    end
+
+    def call(date_field = "payment_date")
+      @date_field = date_field
```

Comment:
**nit** - feel free to ignore, I personally prefer `date_attribute` for naming and use of symbol `:payment_date` as default